tests/test_push.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
Not that I mind, but the license headers aren't technically needed per-file here. Also tests are generally just dedicated to the public domain anyway.